machine_manager/views/product_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="product_normal_form_view_inh" model="ir.ui.view">
```

Comment:
@pedrobaeza In general, can I change Ids when porting modules to OCA? Or do you have these modules in a production environment?